examples/model_simulated/graphite_thermistor_simulated.py

```diff
@@ -130,6 +130,9 @@ def get_thermistor_temperature():
                                port=config['GraphitePort']))
 
     graphite_reg_dev = graphite.register(thermistor_model)
+    if graphite_reg_dev is None:
+        print "Device registration to Graphite failed"
+        exit()
```

Comment:
I prefer something like 
if graphite_reg_dev:
       do all the stuff here
else:
      report error